workflow-generator/src/main/java/com/wipro/ats/bdre/wgen/HiveActionNode.java

```diff
@@ -85,7 +85,7 @@ public String getXML() {
                 "                <value>${wf:actionData(\"init-job\")[\"min-batch-id-map." + getId() + "\"]}</value>\n" +
                 "            </property>\n" +
 //TODO:Fix lineage and enable it
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1135)